airbyte-webapp/src/views/Connector/ServiceForm/components/WarningMessage.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { FormattedHTMLMessage } from "react-intl";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+
+const Content = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 13px 20px;
+  border: 1px solid ${({ theme }) => theme.redColor};
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 18px;
+  white-space: break-spaces;
+  margin-top: 16px;
+`;
+
+const Exclamation = styled(FontAwesomeIcon)`
+  font-size: 20px;
+  margin-right: 12px;
+  color: ${({ theme }) => theme.redColor};
+`;
+
+type WarningMessageProps = {
+  stage: string;
```

Comment:
This should accept not an arbitrary `string`, but more specifically just the ALPHA and BETA values, that we also have a message for:

```suggestion
  stage: ReleaseStage.ALPHA | ReleaseStage.BETA;
```

Reply:
Good catch